src/expr/src/scalar/func.rs

```diff
@@ -2076,8 +2076,15 @@ fn regexp_split_to_array<'a>(
     let regexp = regexp.unwrap_str();
     let flags = flags.unwrap_str();
     let regexp = build_regex(regexp, flags)?;
-    let found = mz_regexp::regexp_split_to_array(text, &regexp);
+    regexp_split_to_array_re(text, &regexp, temp_storage)
+}
 
+fn regexp_split_to_array_re<'a>(
```

Comment:
What does "re" mean?

Reply:
It accepts a Regex argument? I don't like the name choice here but don't care that much either.